cmd/podman/varlink/io.podman.varlink

```diff
@@ -658,8 +658,13 @@ method PauseContainer(name: string) -> (container: string)
 # See also [PauseContainer](#PauseContainer).
 method UnpauseContainer(name: string) -> (container: string)
 
+# Attach takes the name or ID of a container and sets up a the ability to remotely attach to its console.
 method Attach(name: string) -> ()
```

Comment:
Attach(name: string, start: bool) would be a preferred API. Not two endpoints. Service side can have two code paths as needed.